docs/source/index.rst

```diff
@@ -1,7 +1,7 @@
 cleanlab documentation
 ======================
 
-`cleanlab <https://github.com/cleanlab/cleanlab>`_ **automatically finds and fixes errors in your ML datasets.**
+`cleanlab <https://github.com/cleanlab/cleanlab>`_ **automatically finds and fixes label issues in your ML datasets.**
 
 | This reduces manual work needed to fix data issues and helps train reliable ML models on noisy real-world datasets. ``cleanlab`` has already found thousands of `label errors <https://labelerrors.com>`_ in ImageNet, MNIST, and other popular ML benchmarking datasets, so let's get started with yours!
```

Comment:
[Nit] If we go with the above wording, should change this to reduce redundancy:
"to fix data issues"
-> 
"to fix data errors"

Reply:
i've actually changed the above wording to "label issues" too. do you mean we should change "to fix data issues" to "to fix data label issues"?

Reply:
No I am saying: Keep above as is.

Change: "to fix data issues"
->
"to fix data errors"

Reply:
done!